plugins/services/src/js/structs/PodInstance.js

```diff
@@ -149,4 +149,10 @@ module.exports = class PodInstance extends Item {
   isTerminating() {
     return this.getStatus() === PodInstanceState.TERMINAL;
   }
+
+  isDeleting() {
```

Comment:
This naming isn't ideal because it only affects SDK service. Change name to `isSDKDeleting()`?